skills/dff_voice_skill/scenario/condition.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from df_engine.core import Context, Actor
+import common.dff.integration.context as int_ctx
+
+logger = logging.getLogger(__name__)
+
+
+def voice_message_detected(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
+    voice = int_ctx.get_last_human_utterance(ctx, actor).get("annotations", {}).get("voice_service", {})
+    logger.debug(f"CONDITION.PY VOICE: {voice}")
+    if voice is not None:
```

Comment:
default value is `{}` actually - not none

Reply:
Changed